src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -111,6 +111,9 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
     this.$element.addClass("loading");
     if (this.data.locales) {
       this.$element.addClass(this.data.locales[0].name.toLowerCase());
+      this.$element.prop("lang", this.data.locales[0].name.substring(0,2));
+    }else{
```

Comment:
Minor style suggestion:

```suggestion
    } else {
```